README.md

```diff
@@ -5,7 +5,7 @@ eigensdk-rs is an initiative for rust developers to build AVSs on eigenlayer.
 ## Installation
 
  ```bash
-cargo install eigensdk --features full
```

Comment:
this is the main readme , it is correct.
you. can import cli from eigensdk crate itself

Reply:
have you run `cargo install eigensdk --features full` from a terminal? I get
```
$ cargo install eigensdk --features full

    Updating crates.io index
  Downloaded eigensdk v0.1.0
  Downloaded 1 crate (5.1 KB) in 0.32s
error: there is nothing to install in `eigensdk v0.1.0`, because it has no binaries
`cargo install` is only for installing programs, and can't be used with libraries.
To use a library crate, add it as a dependency to a Cargo project with `cargo add`.
```

Reply:
ah, it should have been cargo add instead of cargo install

Reply:
can you change the instruction here? so we can merge it